columnflow/production/cms/btag.py

```diff
@@ -44,12 +48,13 @@ def btag_weights(
     *get_btag_file* can be adapted in a subclass in case it is stored differently in the external
     files.
 
-    The name of the correction set as well as a list of JEC uncertainty sources which should be
-    propagated through the weight calculation should be given as an auxiliary entry in the config:
+    The name of the correction set, a list of JEC uncertainty sources which should be
+    propagated through the weight calculation, and the column used for b-tagging should
+    be given as an auxiliary entry in the config:
```

Comment:
This does not cover your automatic handling of the column name needed for a given correction. Can you ammend this either in this docstring or the docstring of `get_b_score_column`?